hack/local-image.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Get the name of the operating system using the uname command
+os_name=$(uname)
+set -e
+
+function main() {
```

Comment:
It looks like there is a circular dependency between `Makefile` and `local-image.sh` which is not good, we'd better implement the logic of l`ocal-image.sh` directly in `Makefile`

Reply:
ok, it sounds good ,I misunderstand `hack/verify-boilerplate.go` before,so I will patch this commit later, thanks.